src/components/clusters/NewClusterPage.tsx

```diff
@@ -44,6 +48,7 @@ type NewClusterFormProps = {
 
 const NewClusterForm: React.FC<NewClusterFormProps> = ({ pullSecret = '', versions }) => {
   const { addAlert, clearAlerts } = React.useContext(AlertsContext);
+  const { isFeatureEnabled } = React.useContext(FeatureGateContext);
```

Comment:
```
const isSingleNodeEnabled = useFeature('ASSISTED_INSTALLER_SNO_FEATURE')
```
would be a nicer API.

this hook could be exported from FeatureGateContext module. See HostDialogsContext.tsx for simmilar solution.